Sources/BraveWallet/Crypto/Stores/CryptoStore.swift

```diff
@@ -360,6 +365,9 @@ public class CryptoStore: ObservableObject, WalletObserverStore {
     marketStore.tearDown()
     settingsStore.tearDown()
     
+    // user asset manager
+    userAssetManager.tearDown()
```

Comment:
Need to call `setupObservers()` in `CryptoStore` also or we will miss all updates after memory leak on iOS 17.0..<17.2

Reply:
nice catch. added in [bac1398](https://github.com/brave/brave-ios/pull/8726/commits/bac1398b1fafcbf912160a8e55743509f0eae33f)